admin/class-yoast-dashboard-widget.php

```diff
@@ -38,6 +38,7 @@ public function __construct( WPSEO_Statistics $statistics = null ) {
 	 * Adds dashboard widget to WordPress
 	 */
 	public function add_dashboard_widget() {
+		add_filter( 'postbox_classes_dashboard_wpseo-dashboard-overview', array( $this, 'wpseo_dashboard_overview_class' ) );
```

Comment:
Your explanation makes sense, but it somehow doesn't look right to have a PHP solution for a styling problem.

Wouldn't it make more sense to have all styles for the `yoast` class early in our CSS and then override it later using a `yoast-dashboard-widget` class?